target/linux/ath79/dts/qca9556_avm_fritzdvbc.dts

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include <dt-bindings/gpio/gpio.h>
+#include "qca9556_avm_fritz-repeater.dtsi"
+
+/ {
+	compatible = "avm,fritzdvbc", "qca,qca9556";
```

Comment:
I personally don't like these abbreviated names. I'd go for something really resembling the device name, e.g. avm,fritz-repeater-dvb-c ...

Reply:
Yes, however this is currently the case for all those boards. AVM also changed the name of their scorpion repeater 2 times so I'm fine with that as well as the machine name here.

Reply:
I'm aware and I'm not too fond of that fritz-exception to naming in ath79. However, life's hard and I won't force anybody to change that now for a new device, as your argument is obviously valid.
Despite, it would be ugly to resemble that fritz!wlan naming anyway ...